core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250211_add_workspace_spend_report_table.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="dummy" author="sehsan" id="create_WORKSPACE_SPEND_REPORT_TABLE">
+        <createTable tableName="WORKSPACE_SPEND_REPORT">
+            <column name="GOOGLE_PROJECT_ID" type="VARCHAR(254)">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="PK_WORKSPACE_SPEND_REPORT"/>
+            </column>
```

Comment:
an alternative to the composite primary key is to add an autoincrement bigint `id` column as the primary key, then add a unique constraint/index for the GOOGLE_PROJECT_ID/REPORT_START_DATE/REPORT_END_DATE combination.

Will we ever need some other table to have a foreign key to this WORKSPACE_SPEND_REPORT table? If so, the single `id` column seems better. And of course we can always add the id column later.